code/default/xx_net.sh

```diff
@@ -58,8 +60,6 @@ if [ "$(id -u)" != "0" ]; then
     exit 0
 fi
 
-# `readlink -f` won't work on Mac, this hack should work on all systems.
-cd $("${PYTHON}" -c "import os; print os.path.dirname(os.path.realpath('$0'))")
```

Comment:
之前cd的方法在注册为服务时，启动时有问题，但是用service xx_net start又能启动，后面把nohub 那句改成调用start.py 就ok了。





On Tue, Jul 12, 2016 at 10:11 PM +0800, "Cat73" <notifications@github.com> wrote:





> @@ -58,8 +60,6 @@ if [ "$(id -u)" != "0" ]; then
>      exit 0
>  fi
>
> -# `readlink -f` won't work on Mac, this hack should work on all systems.
> -cd $("${PYTHON}" -c "import os; print os.path.dirname(os.path.realpath('$0'))")

这里建议改为`cd "${PACKAGE_PATH}"`

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/XX-net/XX-Net/pull/3844/files/7c9ea1575c21c5d6f66cca870b92a85189cebeae#r70445570